sed/loader/flash/buffer_handler.py

```diff
@@ -12,11 +11,70 @@
 from sed.core.dfops import forward_fill_lazy
 from sed.loader.flash.dataframe import DataFrameCreator
 from sed.loader.flash.utils import get_channels
-from sed.loader.flash.utils import initialize_paths
+from sed.loader.flash.utils import get_dtypes
 from sed.loader.utils import get_parquet_metadata
 from sed.loader.utils import split_dld_time_from_sector_id
 
 
+DF_TYP = ["electron", "timed"]
+
+
+class BufferFilePaths:
+    """
+    A class for handling the paths to the raw and buffer files of electron and timed dataframes.
+    A list of file sets (dict) are created for each H5 file containing the paths to the raw file
+    and the electron and timed buffer files.
+
+    Structure of the file sets:
+    {
+        "raw": Path to the H5 file,
+        "electron": Path to the electron buffer file,
+        "timed": Path to the timed buffer file,
```

Comment:
Is it really necessary to store the data twice (actually three times)? This appears very inefficient to me. Why is this necessary?

Reply:
I'd consider the buffer files the raw files for our case as the flash raw h5 files need to be restructured and filled. 

Of course seperately storing electron and timed data seems redundant. But like we saw from your benchmarks, dask seems to perform rather poorly when the indexes are not monotonic. And they aren't if we drop nans after loading the data. 

Moreover, this now gives us correct counts from the parquet metadata. And that data, we'd definitely like to use later.

Reply:
fine.